core/src/integration-test/java/com/scalar/db/storage/jdbc/JdbcAdminImportTestUtils.java

```diff
@@ -0,0 +1,442 @@
+package com.scalar.db.storage.jdbc;
+
+import com.scalar.db.api.TableMetadata;
+import com.scalar.db.config.DatabaseConfig;
+import com.scalar.db.io.DataType;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.stream.Collectors;
+import org.apache.commons.dbcp2.BasicDataSource;
+
+public class JdbcAdminImportTestUtils {
+  static final List<String> UNSUPPORTED_DATA_TYPES_MYSQL =
+      Arrays.asList(
+          "BIGINT UNSIGNED",
+          "BIT(8)",
+          "DATE",
+          "DATETIME",
+          "DECIMAL(8,2)",
+          "ENUM('a','b')",
+          "SET('a','b')",
+          "GEOMETRY",
+          "JSON", // we remove this for MariaDB because it is an alias of a supported type, LONGTEXT
+          "NUMERIC",
+          "TIME",
+          "TIMESTAMP",
+          "YEAR");
+  static final List<String> UNSUPPORTED_DATA_TYPES_PGSQL =
+      Arrays.asList(
+          "bigserial",
+          "bit(8)",
+          "bit varying(8)",
+          "box",
+          "cidr",
+          "circle",
+          "date",
+          "inet",
+          "interval",
+          "json",
+          "jsonb",
+          "line",
+          "lseg",
+          "macaddr",
+          "macaddr8",
+          "money",
+          "numeric(8,2)",
+          "path",
+          "pg_lsn",
+          "pg_snapshot", // after v14
+          "point",
+          "polygon",
+          "smallserial",
+          "time",
+          "time with time zone",
+          "timestamp",
+          "timestamp with time zone",
+          "tsquery",
+          "tsvector",
+          "txid_snapshot",
+          "uuid",
+          "xml");
+  static final List<String> UNSUPPORTED_DATA_TYPES_ORACLE =
+      Arrays.asList(
+          "BFILE",
+          "DATE",
+          "FLOAT(54)",
```

Comment:
This is for https://github.com/scalar-labs/scalardb/pull/931/files#diff-de99a8d73f3719cd7ae87026fbfbf7a953774fa4719bfdb18f898b5925e7d484R236? So how about testing with `FLOAT(52)` to test another validation?

Reply:
Oops. The validation is just for logging. Please ignore it...